projects/archiver/src/utils/backend-client.ts

```diff
@@ -63,13 +65,3 @@ export const getImage = async (
 
     return [path, await maybeResponse.buffer()]
 }
-
-export const getColours = async (
```

Comment:
Was this removed on purpose? Seems to not be removed anywhere else.

Reply:
It turned out not to be used anywhere, so I got rid of it as it was in the way when making the PR.